demos/scenario_5_events/events_ini.xml

```diff
@@ -59,9 +59,9 @@ xsi:noNamespaceSchemaLocation="http://xsd.jupedsim.org/0.6/jps_ini_core.xsd">
   </routing>
 
   <!--persons information and distribution -->
-  <agents operational_model_id="2">
```

Comment:
Why did you change router and model here?

Reply:
Operational model 2 is no longer supported. And router is changed back to `global_shortest`.